### PR TITLE
Autofill disable in applicant info and names capture

### PR DESCRIPTION
**Pull request by lekshmimallika-aot**
*Issue #:* /bcgov/entity#5508

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

File: client/src/components/common/applicant-info-1.vue [hunk 1]
```diff
@@ -19,7 +19,7 @@
                             height="50"
                             hide-details="auto"
                             id="lastname"
-                            name="lastname"
+                            :name="Math.random()"
```
Comment (severinbeauvais) [inline client/src/components/common/applicant-info-1.vue:22]: I'm testing this locally and it's still allowing me to "fill the form" using values saved in Chrome.

Can you look into creating a Vue directive for this?
https://vuejs.org/v2/guide/custom-directive.html

**Comment by lekshmimallika-aot** [comment]
The workaround in https://github.com/vuetifyjs/vuetify/issues/2792 is what I used

**Review by cameron-freshworks — COMMENTED** [review]
I don't actually see any auto-population or google help in the form, though i'm 100% sure i did before either.

Is this issue browser settings dependent?

**Review by cameron-freshworks — APPROVED** [review]

**Comment by lekshmimallika-aot** [comment]
As discussed, going to put this in DEV and see whether it is working for anyoneelse

**Pull request merged**